demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -110,9 +110,10 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
         };
         if (mode === "edit") {
             if (!id) throw new Error();
+            const { ...updateInput } = output;
             await client.mutate<GQLUpdateProductMutation, GQLUpdateProductMutationVariables>({
                 mutation: updateProductMutation,
-                variables: { id, input: output },
+                variables: { id, input: updateInput },
```

Comment:
It would be good to have a concrete example in the product form. Otherwise this change seems unnecessary. Maybe we could make one field readonly?